CWE/events/italy.txt

```diff
@@ -455,6 +455,1450 @@ country_event = {
+		war = no
+		
+		#Austria exists
+		OR = { 
+			exists = AUS 
+			exists = ROA 
+		}
+		
+		#ITA owns South Tyrol
+		is_possible_vassal = SUT
+		
+		#Location of incident
+		owns = 735
+	}
+
+	mean_time_to_happen = { months = 6 }
+
+	option = {
+		name = "Persecute the terrorists"
+		AUS_734 = {
+			flashpoint_tension = 25
+			any_pop = { 
+				limit = { OR = { culture = german culture = austrian_german } }
+				ideology = { factor = 0.05 value = nationalist } 
+				militancy = 5
+			}
+			any_pop = { 
+				limit = { OR = { culture = italian culture = padanian } }
+				consciousness = 5
+			}
+		}
+		
+		735 = { add_province_modifier = { name = terrorist_chaos duration = 183 } } # Bozen
+		
+		set_country_flag = ITA_1961_Feuernacht
+		
+		ai_chance = { factor = 0.8 }
+	}
+	
+	option = {
+		name = "Persecute the terrorists and disband the SVP"
+		AUS_734 = {
+			flashpoint_tension = 35
+			any_pop = { 
+				limit = { OR = { culture = german culture = austrian_german } }
+				ideology = { factor = 0.1 value = nationalist } 
+				militancy = 7
+				consciousness = 4
+			}
+			any_pop = { 
+				limit = { OR = { culture = italian culture = padanian } }
+				consciousness = 3
+			}
+		}
+		
+		735 = { add_province_modifier = { name = terrorist_chaos duration = 183 } } # Bozen
+		
+		country_event = 8020324 
+		
+		ai_chance = { factor = 0.1 }
+	}
+	
+	option = {
+		name = "Cede to the terrorists"
+		AUS_734 = {
+			any_pop = { 
+				limit = { OR = { culture = italian culture = padanian } }
+				ideology = { factor = 0.1 value = nationalist } 
+				militancy = 2
+				consciousness = 4
+			}
+		}
+		
+		random_owned = {
+			limit = { owner = { exists = ROA } }
+			owner = {
+				any_owned = {
+					limit = { province_id = 735 }
+					add_core = AUS
+					secede_province = ROA
+				}
+			}
+		}
+		random_owned = {
+			limit = { owner = { ROA = { exists = no } } }
+			owner = {
+				any_owned = {
+					limit = { province_id = 735 }
+					add_core = AUS
+					secede_province = AUS
+				}
+			}
+		}
+		
+		ai_chance = { factor = 0.1 }
+	}
+
+}
+
+#Alto-Adige's Long Summer
+country_event = {
+  	id = 8020324
+  	title = "Alto-Adige's Long Summer"
+  	desc = EVT_8020324_DESC
+  	picture = "ITA_Long_Summer"
+	
+  	is_triggered_only = yes
+
+	option = {
+		name = "Legalize the SVP"
+		AUS_734 = {
+			any_pop = { 
+				limit = { OR = { culture = german culture = austrian_german } }
+				ideology = { factor = 0.04 value = conservative } 
+				ideology = { factor = 0.03 value = big_tent } 
+				militancy = -3
+				consciousness = 2
+			}
+			any_pop = { 
+				limit = { OR = { culture = italian culture = padanian } }
+				scaled_militancy = { factor = 4 ideology = nationalist }
+                scaled_consciousness = { factor = 2 ideology = nationalist }
+				militancy = 2
+				consciousness = 1
+			}
+		}
+		
+		734 = { add_province_modifier = { name = terrorist_chaos duration = 183 } } 
+		
+		random_country = { limit = { owns = 612 } country_event = 8020325 } 
```

Comment:
Must check to make sure the owner is AUS or ROA in case the allied occupation continues longer than expected.